app/controllers/keywords_controller.rb

```diff
@@ -1,31 +1,25 @@
 # frozen_string_literal: true
 
-require 'csv'
-
 class KeywordsController < ApplicationController
   before_action :set_keyword, only: %i[search_result]
 
-  MAX_KEYWORDS_ALLOWED = 100
   def new; end
 
   def create
     if keyword_params[:file].present? && keyword_params[:file].content_type == 'text/csv'
```

Comment:
These checks could be moved into `CsvManager::CsvProcessor` as well. So the validity of the CSV file would depend:
- Is the file valid
- Is the file in the requested format
- Does the file have less than 100 keywords